### PR TITLE
Allow configuring the controller feature gates in helm chart

### PR DESCRIPTION
**Pull request by khizunov**
### Issue
#2775 

### Description
Allow configuring the controller feature gates in helm chart.
Fix link to documentation.

### Checklist
- [ ] Added tests that cover your change (if possible)
- [x] Added/modified documentation as required (such as the `README.md`, or the `docs` directory)
- [x] Manually tested
- [x] Made sure the title of the PR is a good description that can go into the release notes

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: helm/aws-load-balancer-controller/README.md [hunk 1]
```diff
@@ -163,85 +163,86 @@ Chart release v1.2.0 and later enables high availability configuration by defaul
 ## Configuration
 
 The following tables lists the configurable parameters of the chart and their default values.
-The default values set by the application itself can be confirmed [here](https://kubernetes-sigs.github.io/aws-load-balancer-controller/guide/controller/configurations/).
```
Comment (kishorj) [inline helm/aws-load-balancer-controller/README.md:166]: curious if this huge diff is due to auto-formatting. If not, lets restrict the change to the newly added row.

File: helm/aws-load-balancer-controller/values.yaml [hunk 2]
```diff
@@ -284,6 +284,11 @@ backendSecurityGroup:
 # disableRestrictedSecurityGroupRules specifies whether to disable creating port-range restricted security group rules for traffic
 disableRestrictedSecurityGroupRules:
 
+# featureGates set of key: value pairs that describe AWS load balance controller features
+featureGates: {}
```
Comment (kishorj) [inline helm/aws-load-balancer-controller/values.yaml:288]: Lets group under controllerConfig, we plan to group together related values in future, we will start with the new configurations. Something like this:
 
```
# controllerConfig specifies controller configuration
controllerConfig:
    # featureGates set of key: value ...
    featureGates: {}
        # ...
```

File: helm/aws-load-balancer-controller/README.md (hunk 1 above)
Comment (khizunov) [reply helm/aws-load-balancer-controller/README.md:166]: fixed, was some IDE auto formatting

File: helm/aws-load-balancer-controller/values.yaml (hunk 2 above)
Comment (khizunov) [reply helm/aws-load-balancer-controller/values.yaml]: Done

Comment (khizunov) [reply helm/aws-load-balancer-controller/values.yaml]: @kishorj

**Review by kishorj — APPROVED** [review]

**Pull request merged**